transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/ForcedSender.java

```diff
@@ -26,16 +25,11 @@
 public final class ForcedSender {
   @WorkerThread
   public static void sendBlocking(Transport<?> transport, Priority priority) {
-    @SuppressLint("DiscouragedApi")
-    TransportContext context = getTransportContextOrThrow(transport).withPriority(priority);
-    TransportRuntime.getInstance().getUploader().logAndUpdateState(context, 1);
-  }
-
-  private static TransportContext getTransportContextOrThrow(Transport<?> transport) {
     if (transport instanceof TransportImpl) {
-      return ((TransportImpl<?>) transport).getTransportContext();
+      TransportContext context =
+          ((TransportImpl<?>) transport).getTransportContext().withPriority(priority);
+      TransportRuntime.getInstance().getUploader().logAndUpdateState(context, 1);
     }
-    throw new IllegalArgumentException("Expected instance of TransportImpl.");
```

Comment:
Let's add a warning log statement here, there's logging util available in this sdk under
`com.google.android.datatransport.runtime.logging.Logging`